src/ui/components/ProductList.tsx

```diff
@@ -0,0 +1,21 @@
+import { ProductElement } from "./ProductElement";
```

Comment:
This file is already on `canary` so how come it shows as a completely new here?
Bad rebase?